extensions/wikia/CommunityPage/CommunityPage.i18n.php

```diff
@@ -63,14 +63,14 @@
 	'communitypage-no-members' => 'This community has no members',
 	'communitypage-help-us-grow' => "Help us grow '''{{SITENAME}}'''!",
 	'communitypage-new' => 'New',
-	'communitypage-popularpages-title' => 'Trending pages',
-	'communitypage-popularpages-description' => 'These pages are gaining traction with your readers! Make sure they are up to your standards and have all the latest images and information.',
-	'communitypage-uncategorizedpages-title' => 'Pages without categories',
-	'communitypage-uncategorizedpages-description' => 'Add categories to these pages. Good categorization is vital to a successfully organized wiki!',
+	'communitypage-popularpages-title' => 'Please proofread these pages',
+	'communitypage-popularpages-edittext' => 'Read and fix',
+	'communitypage-uncategorizedpages-title' => 'Please add categories to these pages',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "communitypage-uncategorizedpages-title" 13 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)